weblab_cg/_printer.py

```diff
@@ -61,34 +61,36 @@ class Printer(sympy.printing.printer.Printer):
         A function that converts derivatives to strings.
 
     """
-    # List of functions handled by python's `math` module
+    # Dictionary of functions handled by python's `math` module and their corresponding names to be generated
+    # or a lambda expression to generate the name
     _math_functions = {
-        'acos',
-        'acosh',
-        'asin',
-        'asinh',
-        'atan',
-        'atan2',
-        'atanh',
-        'ceil',     # called ceiling in sympy
-        'cos',
-        'cosh',
-        'exp',
-        'expm1',
-        'factorial',
-        'floor',
-        'log',
-        'log10',
-        'log1p',
-        'log2',
-        'sin',
-        'sinh',
-        'tan',
-        'tanh',
+        'Abs': (lambda x: 'abs' if x.replace('-', '').isnumeric() else 'fabs'),
```

Comment:
??? Why not just map it to `abs` ? And map the rest to `math.acos` etc.?

Reply:
because it will fail if the number is not an in that's why we could just map it to fabs of course

Reply:
The math.abs and math.fabs methods change the type of the output, which I'm not sure we always want. Seems easier to just use the [built-in `abs` method](https://docs.python.org/3.5/library/functions.html#abs)

Reply:
what built-in abs method are you talking about?